tooltester/.tooltester/config

```diff
@@ -0,0 +1,7 @@
+server-url = https://www.dockstore.org:8443
+token = ""
+jenkins-server-url = http://172.18.0.22:8080/
+jenkins-username = travis
```

Comment:
change these passwords so that they are clearly not real (ex: `fake_password` or similar)